src/ClientData/DataManagerTest.cpp

```diff
@@ -115,6 +135,56 @@ TEST(DataManager, CanOnlyBeUsedFromTheMainThread) {
                                             WineSyscallHandlingMethod::kNoSpecialHandling);
   CallMethodOnDifferentThreadAndExpectDeath(data_manager,
                                             &DataManager::wine_syscall_handling_method);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::GetSelectionTimeRange);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::ClearSelectionTimeRange);
+  CallMethodOnDifferentThreadAndExpectDeath(data_manager, &DataManager::SetSelectionTimeRange,
+                                            TimeRange(kStart, kEnd));
+}
+
+TEST(DataManager, SelectionTimeRangeDefaultValue) {
+  DataManager data_manager;
+  EXPECT_EQ(data_manager.GetSelectionTimeRange().has_value(), false);
+}
+
+TEST(DataManager, SetSelectionTimeRange) {
+  DataManager data_manager;
+  data_manager.SetSelectionTimeRange(TimeRange(kStart, kEnd));
+  EXPECT_EQ(data_manager.GetSelectionTimeRange().has_value(), true);
```

Comment:
General comments: it should be `ASSERT_*`, because the checks below make no sense if this one fails. And it should be `ASSERT_TRUE`.

But in this case I suggest to implement a matcher (with `MATCHER_P2`) for the `TimeRange`, so here we could just write 

```
EXPECT_THAT(data_manager.GetSelectionTimeRange(), Optional(TimeRangeEq(kStart, kEnd)));
```

Moreover, the matcher could be nicely reused in the tests below.

Reply:
done